.github/workflows/not-python.yml

```diff
@@ -2,6 +2,17 @@ name: Run Python Linting and Unit Tests
 
 on:
   push:
+    branches-ignore:
+      - master
+    paths-ignore:
+      - 'client/python/**'
+      - 'build/python-client/**'
+      - 'pkg/api/*.proto'
+      - '.github/workflows/python.yml'
+      - 'docs/python_armada_client.md'
+  pull_request:
+    branches:
+      - master
```

Comment:
this needs to be updated to branches-ignore, like in the go.yml, and I missed this several places